sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -986,6 +1072,17 @@ class HistGradientBoostingClassifier(BaseHistGradientBoosting,
         and 0 respectively correspond to a positive constraint, negative
         constraint and no constraint. Read more in the :ref:`User Guide
         <monotonic_cst_gbdt>`.
+    categorical : array-like of bool of shape (n_features) or 'pandas', \
+        default=None.
+        Indicates the categorical features.
```

Comment:
haven't checked but might require a skipline

Reply:
It looks good:
<img width="799" alt="Capture d’écran 2020-04-14 à 10 49 09" src="https://user-images.githubusercontent.com/17053934/79204859-80f70a80-7e3d-11ea-8df7-0f77a870381c.png">